api/hasura/actions/_handlers/createEpoch.ts

```diff
@@ -128,7 +129,7 @@ async function handler(request: VercelRequest, response: VercelResponse) {
     return;
   }
 
-  insertNewEpoch(response, input);
+  insertNewEpoch(response, input, creatorProfileId);
```

Comment:
this is kinda messy, but the input type is what comes from the FE, so don't want to add to that... curious if there's a better paradigm.